src/map/skill.cpp

```diff
@@ -4138,8 +4138,6 @@ int64 skill_attack (int attack_type, struct block_list* src, struct block_list *
 				}
 				break;
 		}
-		if( sd )
-			skill_onskillusage(sd, bl, skill_id, tick);
```

Comment:
This solves the problem, but seems to generate another one.
This bonus has stopped working:

```

                     bonus4 bAutoSpellOnSkill,"RK_DRAGONBREATH_WATER","NPC_ACIDBREATH",getskilllv("RK_DRAGONBREATH_WATER"),1000;
                     bonus4 bAutoSpellOnSkill,"RK_DRAGONBREATH","NPC_THUNDERBREATH",getskilllv("RK_DRAGONBREATH"),1000;
                     
```
Probably a skill calling another skill of a different type fails?